storage/storage-interface/src/lib.rs

```diff
@@ -35,9 +35,9 @@ use serde::{Deserialize, Serialize};
 use std::{collections::HashMap, sync::Arc};
 use thiserror::Error;
 
+pub mod async_proof_fetcher;
 pub mod cached_state_view;
 mod executed_trees;
-#[cfg(any(feature = "testing", feature = "fuzzing"))]
```

Comment:
why?

Reply:
Compiler complains it can not ```use crate::mock::MockReaderWriter``` if I keep this line.